ob-pattern-2/templates/obbi-dashboard/wso2ob-pattern-2-obbi-dashboard-service.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: wso2ob-pattern-2-obbi-dashboard-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    deployment: wso2ob-pattern-2-obbi-dashboard
+  ports:
+  - name: https
+    port: 9643
+    targetPort: 9643
+    protocol: TCP
+  - name: rest-api-port-1
+    port: 9449
+    targetPort: 9449
+    protocol: TCP
+  - name: rest-api-port-2
+    port: 9713
+    targetPort: 9713
+    protocol: TCP
+  - name: rest-api-port-3
+    port: 9613
+    targetPort: 9613
+    protocol: TCP
+  - name: rest-api-port-4
+    port: 7713
+    targetPort: 7713
+    protocol: TCP
+  - name: rest-api-port-5
+    port: 7613
+    targetPort: 7613
+    protocol: TCP
+  - name: rest-api-port-6
+    port: 8006
+    targetPort: 8006
+    protocol: TCP
+
```

Comment:
@kaumini remove the unnecessary, additional EOF lines. Applies to some other files as well.

Reply:
Fixed with edf1bb86b4ea94c37158885b59e738d865e2347a